backend/src/main/java/com/googleinterns/smb/CartDocumentsController.java

```diff
@@ -0,0 +1,49 @@
+package com.googleinterns.smb;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+
+import org.springframework.web.bind.annotation.*;
+
+class ItemDetails{
+	public String userID;
+	public String itemName;
+	public int quantity = 0;
+
+	@Override
+	public String toString(){
+		return userID;
+	}
+}
```

Comment:
Move data class as separate class and move all the **public** fields to **private** and provide the getters/setters for the same.

Reply:
Done